wallet/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Introduction
+---
+
+# Integrate your dapp with the MetaMask wallet
+
+MetaMask provides powerful tools for connecting dapps to wallets. With **MetaMask SDK**, developers can easily create seamless wallet connections across desktop and mobile platforms. The **Wallet API** offers flexibility in interacting with wallets and blockchain networks via standardized JSON-RPC calls, enabling direct access to key functionalities.
+
+Get started by [connecting to MetaMask](connect/index.md).
+
+## Why use MetaMask SDK?
+
+The **MetaMask SDK** allows you to establish a reliable connection between your dapp and MetaMask browser extension or MetaMask Mobile, providing cross-platform compatibility with a consistent user experience. Key benefits include:
```

Comment:
We should clarify if it's called "The MetaMask SDK" or "MetaMask SDK." Current docs omit "The."